assets/src/components/UserSettingSnackbar.js

```diff
@@ -25,14 +30,21 @@ function UserSettingSnackbar (props) {
     }
   }, [saved])
 
+  const snackbarDuration = Math.max(snackbarMessage.length * 200, 4000)
+  
   return (
     <Snackbar
       anchorOrigin={{
         vertical: 'bottom',
         horizontal: 'left'
       }}
+      ContentProps={{
+        'role': 'alertdialog',
```

Comment:
This can be just `role: 'alertdialog`, since you're creating an object; I think `standard` prefers unquoted property names.

Reply:
fixed